internal/cmd/local/command_broker_update.go

```diff
@@ -0,0 +1,103 @@
+package local
+
+import (
+	"context"
+
+	"github.com/antihax/optional"
+	"github.com/spf13/cobra"
+
+	"github.com/confluentinc/cli/internal/pkg/broker"
+	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/examples"
+	"github.com/confluentinc/cli/internal/pkg/kafkarest"
+	"github.com/confluentinc/cli/internal/pkg/output"
+	"github.com/confluentinc/cli/internal/pkg/properties"
+	"github.com/confluentinc/kafka-rest-sdk-go/kafkarestv3"
+)
+
+func (c *Command) newBrokerUpdateCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update [id]",
+		Short: "Update a Confluent local broker configurations.",
```

Comment:
I believe the product "Confluent Local" is a proper noun that should be capitalized in all cases?
```suggestion
		Short: "Update Confluent Local broker configurations.",
```